src/views/components/injected/CourseCatalogInjectedPopup/Description.tsx

```diff
@@ -1,37 +1,87 @@
 import Text from '@views/components/common/Text/Text';
 import clsx from 'clsx';
 import React from 'react';
+import type { Course } from 'src/shared/types/Course';
+import { CourseCatalogScraper } from 'src/views/lib/CourseCatalogScraper';
+import { SiteSupport } from 'src/views/lib/getSiteSupport';
```

Comment:
Use path alias and sort imports.

```suggestion
import type { Course } from '@shared/types/Course';
import { CourseCatalogScraper } from '@views/lib/CourseCatalogScraper';
import { SiteSupport } from '@views/lib/getSiteSupport';
```